understanding/22/page-break-navigation.html

```diff
@@ -24,7 +24,7 @@ <h2>Page Break Navigation Normative Text</h2>
       </blockquote>
 
       <h3>Definitions:</h3>
-      <p><strong>page break locators</strong> programmatic markers that are arranged in a meaningful sequence to determine the location of a page in relation to others in the set.</p>
+      <p><strong>page break locators</strong> programmatically determinable anchors or destinations that are arranged in a meaningful sequence to determine the location of a page in relation to others in the set.</p>
```

Comment:
main WCAG spec document uses `<dl>`/`<dt>`/``<dd>` for definitions - may be worth adopting the same structure here?

Reply:
This part is only in the understanding-doc previews, when it's published properly it is discarded.